base/summarysize.jl

```diff
@@ -6,17 +6,21 @@ struct SummarySize
     frontier_i::Vector{Int}
     exclude::Any
     chargeall::Any
+    count::Bool
 end
 
 nth_pointer_isdefined(obj, i::Int) = ccall(:jl_nth_pointer_isdefined, Cint, (Any, Csize_t), obj, i-1) != 0
 get_nth_pointer(obj, i::Int) = ccall(:jl_get_nth_pointer, Any, (Any, Csize_t), obj, i-1)
 
 """
-    Base.summarysize(obj; exclude=Union{...}, chargeall=Union{...})::Int
+    Base.summarysize(obj; count = false, exclude=Union{...}, chargeall=Union{...})::Int
```

Comment:
This might be clearer as:
```suggestion
    Base.summarysize(obj; count::Symbol = :size, exclude=Union{...}, chargeall=Union{...})::Int
```

so the new functionality would be `summarysize(x; count=:allocations)`